sspl_test/plans/sanity.pln

```diff
@@ -1,16 +1,13 @@
+alerts.self_hw.test_node_ipmitool_func
```

Comment:
why changes in this plan?

Reply:
'self' plan renamed to 'sanity' and existing 'sanity'  test plan renamed to 'dev_sanity.

Reply:
Sanity plan contains all the tests which were present in self plan.